docs/guides/earning-cred.md

```diff
@@ -1,5 +1,35 @@
 ---
 title: 💧 Earning Cred
+description: How to get Cred from your contributions.
 ---
 
-TODO
+As soon as you interact with Sourcecred, you will be part of the Contribution Graph (i.e. a network of connected contributions) in which some [Cred] will flow or be minted in your direction. 
+
+
+Your contributions are tracked across the three active plugins we have for now. The types of contributions & plugins are likely to increase. But for now, small is beautiful.
+
+
+| 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]|
+| -- | -- | -- |
+|someone gives your post : [![](https://sourcecred.io/img/favicon.png)](https://sourcecred.io/img/favicon.png)  or :100: or :+1: emoji | someone gives your post/reply a :heart: | You open a pull request  |
+
+
+
+
+### 💧 Cred Minting
+The community decides to Mint some Cred for you, e.g. because they found something valuable. This Cred just magically appears out of nowhere in your direction. If we compare Cred to water, minting Cred would be like rain - it would fall from the sky, new water collected in your pond !
```

Comment:
I think it'd be helpful to explain how minting works in the three plugins right now:

- On GitHub, every pull request mints Cred
- On Discourse, every :heart: on a forum post mints Cred
- On Discord, certain reactions mint Cred